v21.2/cockroach-sql.md

```diff
@@ -13,6 +13,10 @@ To exit the interactive shell, use `\q`, `quit`, `exit`, or `ctrl-d`.
 If you want to experiment with CockroachDB SQL but do not have a cluster already running, you can use the [`cockroach demo`](cockroach-demo.html) command to open a shell to a temporary, in-memory cluster.
 {{site.data.alerts.end}}
 
+## Prerequisites
+
+The [role option of the user](create-role.html#role-options) logging in must be `SQLLOGIN`. If the user's role option has been set to `SQLNOLOGIN`, the user cannot log in using the SQL CLI with any authentication method.
```

Comment:
`SQLNOLOGIN` should be `NOSQLLOGIN` (the negative is always prepended)

I'm a bit worried about the wording here implying that the user has to *take action* before being able to log into SQL. Is it obvious that they have this role by default? (under the hood, adding the `NOSQLLOGIN` role introduces blocking but the "absence" of this block is taken to affirm that they have `SQLLOGIN`, does that make sense? `LOGIN` and `NOLOGIN` work the same way, the user doesn't have to do anything to have `LOGIN` by default.)

Reply:
Thanks for the catch. I missed that one.

So does the prereq have to be 


The [role option of the user](create-role.html#role-options) logging in must be `SQLLOGIN`, which is granted by default. If the user's role option has been set to `SQLNOLOGIN`, the user cannot log in using the SQL CLI with any authentication method.

Reply:
Correct. It appears we did not have this section previously despite the fact that `LOGIN` and `NOLOGIN` have the same effect on SQL logins so I'm also wondering if this section is necessary. If it is, it should also reference the `LOGIN` role option, I think.

Reply:
I think we should have a section describing required permissions on any function that won't work if the user doesn't have it. IMO, the fact that we didn't have it before is a shortcoming.

Reply:
👍🏻  LGTM thx!